pages/roles.tsx

```diff
@@ -0,0 +1,141 @@
+import { useUser } from '../lib/hooks'
+import { useEffect, useState } from 'react'
+import Layout from '../components/Layout'
+
+async function ubu(url, method, data={}) {
```

Comment:
`ubu` is not a very descriptive function name

all i think of is "sit, ubu, sit. good dog."